WordPressCom-Stats-iOS/UI/NSObject+StatsBundleHelper.m

```diff
@@ -0,0 +1,13 @@
+#import "NSObject+StatsBundleHelper.h"
+
+@implementation NSObject (StatsBundleHelper)
+
+- (NSBundle *)statsBundle
+{
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"WordPressCom-Stats-iOS" ofType:@"bundle"];
```

Comment:
@koke Bundle for class won't work here as the class isn't itself inside of the bundle.